src/Microsoft.TemplateEngine.Edge/LocalizableStrings.resx

```diff
@@ -395,4 +395,8 @@ Template cache will be recreated on the next run.</value>
     <value>'IWorkloadsInfoProvider' component provided by host provided some duplicated workloads (duplicates: {0}). Duplicates will be skipped.</value>
     <comment>{0} is the list of duplicates</comment>
   </data>
-</root>
\ No newline at end of file
+  <data name="TemplatePackageManager_Warning_DetectedTemplatesIdentityConflict" xml:space="preserve">
+    <value>The template {0} installed from package {1} has the identity {2} that conflicts with an already existing template {3}. The template {3} will be overwritten.</value>
```

Comment:
```suggestion
    <value>The template {0} installed from package {1} has the identity {2} that conflicts with an already existing template {3} from package {4}. The latter template will be overwritten.</value>
```
(just in case {0} and {3} are same)